scripts/TerminatePausedAssessment.php

```diff
@@ -131,4 +131,18 @@ protected function addReport($type, $message)
             $message
         ));
     }
+    
+    /**
+     * Return Termination Reasons.
+     * 
+     * Provides the 'reasons' information array with keys 'category' and 'subCategory'.
+     * This method may be overriden by subclasses to provide customer specific information.
+     * 
+     * @return arrray.
+     */
+    protected function getTerminationReasons()
+    {
+        // @fixme remove customer specific information.
+        return ['category' => 'Technical', 'subCategory' => 'ACT'];
```

Comment:
Not related to this story, but it would better to set this content in the proper extension. I don't think the `subCategory` is actually generic :)